source/reference/minio-mc/mc-support-perf.rst

```diff
@@ -0,0 +1,208 @@
+
+#. :mc-cmd:`~mc support perf net`
+
+   Measure the network throughput of all nodes.
+
+Examples
+--------
+
+Measure Speed of an Object
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Measure the speed of an object on the alias ``minio1``.
+MinIO autotunes concurrency to obtain maximum throughput and IOPS (Input/Output Per Second).
+
+.. code-block:: shell
+   :class: copyable
+ 
+   mc support perf object minio1
+
+Measure Speed of an Object of a Specific Size for a Specific Duration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run object speed measurement for 20 seconds with object size of 128MiB on alias ``minio1``.
+MinIO autotunes concurrency to obtain maximum throughput.
+
+.. code-block:: shell
+   :class: copyable
+
+   mc support perf object minio1 --duration 20s --size 128MiB
+
+
+Test Speed of All Drives on All Nodes with Default Specifications
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run drive speed measurements on all drive on all nodes for a cluster with alias ``minio1``.
+The command does not specify the blocksize, so the default of 4MiB is used.
+
+.. code-block:: shell
+   :class: copyable
+ 
+   mc support callhome get minio1
+
+Test Drive Speed Measurements with Custom Specifications
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run drive speed measurements on a cluster with alias ``minio1`` specificying a blocksize of 64KiB and data read/written from each drive of 2GiB.
+
+.. code-block:: shell
+   :class: copyable
+
+   mc support perf drive minio1 --blocksize 64KiB --filesize 2GiB
+
+Test Network Throughput
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Run a network throughput test on a cluster with alias ``minio1``.
+
+.. code-block:: shell
+   :class: copyable
+
+   mc support perf net minio1
+
+Syntax
+------
+
+.. mc-cmd:: drive
+   :fullpath:
+
+Measure the speed of the drives in a cluster.
+
+.. code-block:: shell
+               
+   mc [GLOBAL FLAGS] support perf drive   \
+                   [--concurrent]         \
+                   [--verbose, -v]        \
+                   [--filesize]           \
+                   [--blocksize]          \
+                   [--serial]             \
+                   ALIAS
+
+.. mc-cmd:: object
+   :fullpath:
+
+Measure the speed of reading and writing objects in a cluster.
+
+.. code-block:: shell
+               
+   mc [GLOBAL FLAGS] support perf object  \
+                   [--duration]           \
+                   [--size]               \
+                   [--concurrent]         \
+                   [--verbose, -v]        \
+                   ALIAS  
+            
+.. mc-cmd:: net
+   :fullpath:
+
+Measure the network throughput of all nodes in a cluster.
+
+.. code-block:: shell
+
+   mc [GLOBAL FLAGS] support perf net  \
+                   [--concurrent]      \
+                   [--verbose, -v]     \
+                   [--serial]          \
+                   ALIAS
+
+Parameters
```

Comment:
Hm. if we're doing the params like this here, but above some params only apply to a single command, we should state something like:

```
<parameter> only applies to <command(s)>